sanctions/apps/api/v1/tests/test_views.py

```diff
@@ -27,12 +28,27 @@ def test_sdn_check_missing_args(self):
         response = self.client.post(self.url, HTTP_AUTHORIZATION=self.token)
         assert response.status_code == 400
 
-    # TODO: add test for test_sdn_check_search_fails_uses_fallback
+    @mock.patch('sanctions.apps.api.v1.views.checkSDNFallback')
+    @mock.patch('sanctions.apps.api_client.sdn_client.SDNClient.search')
+    def test_sdn_check_search_fails_uses_fallback(self, mock_search, mock_fallback):
+        mock_search.side_effect = [HTTPError]
+        mock_fallback.return_value = 0
+        response = self.client.post(
+            self.url,
+            content_type='application/json',
+            HTTP_AUTHORIZATION=self.token,
```

Comment:
noticed this with my PR, you can actually drop this HTTP_AUTHORIZATION line and instead just use before you make the post `self.set_jwt_cookie(self.user.id)`

Reply:
It does the same thing? So we probably don't need  the function `generate_jwt_token_header()`

Reply:
you are correct, we need the JWT. `set_jwt_cookie` and `generate_jwt_token_header`  are doing something similar

if you call `set_jwt_cookie`, it generates a JWT payload and then attaches the payload to the test client as a cookie. `generate_jwt_token_header()` will generate the cookie, but then you still have to manually pass the cookie as an AUTH header

i glanced at a few other repos, and i don't actually see `generate_jwt_token_header` used anywhere. while they are functionally equivalent, but i think we should go with the pattern that's used in other places

Reply:
Yes, in some of my tests I used `generate_jwt_token_header()` without `set_jwt_cookie()`, although another I added both 🤷🏻‍♀️ I removed it